src/main/java/io/appium/java_client/AppiumBy.java

```diff
@@ -206,7 +206,7 @@ public static By iOSNsPredicateString(final String iOSNsPredicateString) {
      * @param selector is the value defined to the key attribute of the flutter element
      * @return an instance of {@link AppiumBy.ByFlutterKey}
      */
-    public static By flutterKey(final String selector) {
+    public static FlutterBy flutterKey(final String selector) {
```

Comment:
why are these changes needed?

Reply:
Currenlty all flutter script can work with locatar supported by flutter driver. This change is specifically introduced the simplify the API consumption and to prevent the manual type casting when calling the commands.

```java
driver.waitForInVisible(new WaitParameter().setLocator(AppiumBy.flutterKey("message_field")));
```

Instead of 

```java
driver.waitForInVisible(new WaitParameter().setLocator((AppiumBy.FlutterBy)AppiumBy.flutterKey("message_field")));
```